pkgs/development/libraries/ogre/default.nix

```diff
@@ -8,11 +8,11 @@
 , nvidia_cg_toolkit }:
 
 stdenv.mkDerivation {
-  name = "ogre-1.9.0";
+  name = "ogre-1.9-hg";
```

Comment:
Why the `-hg` suffix? As far as I understand this is the standard Ogre package, if the URL has to be changed it's fine but I think we should leave the package version. Sorry if this is an ignorant comment, I've been out of the loop a while and maybe there's a new rule about this.